### ISSUE TITLE
Improve support for screen readers

### ISSUE THREAD

**Issue by TrechNex** [glimpse-editor/Glimpse#436]
### Is your feature request related to a problem? Please describe
Currently Glimpse Image Editor is based on the GNU Image Manipulation Program 2.10.12 (soon 2.10.18), which uses GTK2.

I have heard mixed feedback about how this works with screen reading software. We should spend time testing this out, and make changes where we can to improve the experience for partially-sighted people.

It is likely that any such improvements would be limited in scope, as we would eventually inherit improves from upstream when they finally release version 3.0 (based on GTK3). But it makes sense for us to investigate how feasible it is to make our own changes.

**Comment by malabo1990** [comment]
I am very glad that you are concerned about visual impairment ... It is very important!

I am from this community.

I want to know what GTK 3 you are working on or create your codes as a programming language D ... I am very sorry that I am not a programmer to help you with ideas.

But I want to understand more clearly how it works in GTK3 for suggestions and ideas.


Thank you.

**Comment by Member1221** [comment]
We're making our own UI toolkit entirely unrelated to GTK 3 for Glimpse NX.
On Windows and macOS I plan to make it plug in to the screen reading apis of those systems, on Linux I'm not entirely sure what we'll do as screen reader support is a general problem in the Linux ecosystem as a whole.

**Comment by malabo1990** [comment]
Well ... I want to tell you that GTK3 is very disadvantages for ipad ... Because visual disability advantages a lot with ipad due to the color of the screen and it includes excellent accessibility from the apple operating system as it has from Affinity Designer and photo .. Its main problem that it is difficult to see cursor ... ipad does not have to worry to get the cursor and all pure touch ... very nice. We should solve this matter.

**Comment by TrechNex** [comment]
Exactly. I make no promises on how much we can do with this as we are working with GTK2 in the fork, and that is around 20 years old! But it is possible we may be able to do some things.

For Glimpse NX (the rewrite) anything is possible at this point, as it is still in the early design stages. 👍 

This issue is specifically for the fork.
